### PR TITLE
3.7-rc.1

### PR DESCRIPTION
**Pull request by bastianallgeier**

### CODE REVIEW

File: i18n/translations/en.json [hunk 1]
```diff
@@ -486,10 +486,10 @@
   "stats.empty": "No reports",
   "system.issues.content": "The content folder seems to be exposed",
   "system.issues.debug": "Debugging must be turned off in production",
-  "system.issues.git": "The .git folder seems to be accessible via the browser",
+  "system.issues.git": "The .git folder seems to be exposed",
   "system.issues.https": "We recommend HTTPS for all your sites",
-  "system.issues.kirby": "The kirby folder seems to be accessible via the browser",
-  "system.issues.site": "The site folder seems to be accessible via the browser",
+  "system.issues.kirby": "The kirby folder seems to be exposed",
+  "system.issues.site": "The site folder seems to be exposed",
```
Comment (lukasbestle) [inline i18n/translations/en.json:492]: Where do these changes come from?

**Comment by afbora** [comment]
Update composer dependencies, please.

**Review by distantnative — DISMISSED** [review]

File: i18n/translations/en.json (hunk 1 above)
Comment (bastianallgeier) [reply i18n/translations/en.json:492]: I made them for this release after seeing that we have mixed wordings there.

**Review by distantnative — APPROVED** [review]

File: i18n/translations/en.json (hunk 1 above)
Comment (lukasbestle) [reply i18n/translations/en.json:492]: Ah, alright. I thought they might be an accidental revert from the Transifex import.

**Review by lukasbestle — APPROVED** [review]

**Pull request merged**